dascore/io/febus/t1utils.py

```
@@ -0,0 +1,90 @@
+
+def _get_coords(fi) -> dc.CoordManager:
+    time_coord = _get_time_coord(fi)
+    distance_coord = _get_distance_coord(fi)
+    dims = ("time", "distance")
+    return get_coord_manager(
+        {"time": time_coord, "distance": distance_coord},
+        dims=dims,
+    )
+
+
+def _get_attrs(path, format, version):
```

Ruff (0.15.15)

[error] Function argument `format` is shadowing a Python builtin

(A002)